<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarActivity</class>
 <widget class="QDialog" name="CalendarActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>371</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FitApp - Календарь активности</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCalendarWidget" name="calendarSets"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="addSet">
             <property name="text">
              <string>Добавить сет</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeSet">
             <property name="text">
              <string>Удалить сет</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="go_backButton">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="SetstableView"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="exerciseLogsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
